Automattic/wp-calypso
# My Sites: remove dead code for feature upsell

#### Changes proposed in this Pull Request

* This PR removes the feature flag `upsell/nudge-a-palooza`, which was disabled in all environments, as well as the code that depends on it.

#### Testing instructions



* Smoke test a build of the site with this code removed to ensure that it builds and does not unexpectedly WSOD as a result of this unused code being removed.

Fixes #44413
